web/src/utils/tcp-drop.ts

```diff
@@ -0,0 +1,296 @@
+/** TODO: remove this file, replaced by https://github.com/netobserv/flowlogs-pipeline/pull/429/files#r1219203830 */
```

Comment:
I keep this file until we validate [full text](https://github.com/netobserv/flowlogs-pipeline/pull/429#discussion_r1219203830) approach

Reply:
is it time now to remove this file ?

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/324/commits/398fa08a3a668788af70b1556cea6cce800cd5b9